pilot/pkg/security/authz/model/model.go

```diff
@@ -29,8 +29,9 @@ const (
 	RBACHTTPFilterName = "envoy.filters.http.rbac"
 
 	// RBACTCPFilterName is the name of the RBAC network filter in envoy.
-	RBACTCPFilterName       = "envoy.filters.network.rbac"
-	RBACTCPFilterStatPrefix = "tcp."
+	RBACTCPFilterName         = "envoy.filters.network.rbac"
+	RBACTCPFilterStatPrefix   = "tcp."
+	RBACShadowRulesStatPrefix = "istio_dry_run_"
```

Comment:
We don't have to differentiate different prefix for tcp and http within dry run?

Reply:
no, we don't need to.